static/app/views/issueDetails/groupEventHeader.tsx

```diff
@@ -26,23 +26,23 @@ function GroupEventHeader({event, group, project}: GroupEventHeaderProps) {
     'related-issues-issue-details-page'
   );
   // This is also called within the TraceTimeline component but caching will save a second call
-  const {isError, isLoading, oneOtherIssueEvent} = useTraceTimelineEvents({
+  const {oneOtherIssueEvent} = useTraceTimelineEvents({
     event,
   });
-  const readyToShow = !isLoading && !isError;
 
   return (
     <StyledDataSection>
       <GroupEventCarousel group={group} event={event} projectSlug={project.slug} />
-      {isRelatedIssuesEnabled && readyToShow && oneOtherIssueEvent === undefined && (
-        <TraceLink event={event} />
-      )}
       {isRelatedIssuesEnabled && oneOtherIssueEvent && (
```

Comment:
If `oneOtherIssueEvent` is defined, we know that the data was fetched and that there was no error.